app/scenes/HomeScreen/reducer.js

```diff
@@ -41,6 +41,7 @@ export const homeContainerReducer = (state = initialState, action) =>
       case homeScreenTypes.REQUEST_FETCH_TRACKS:
         return fetchTracksFromItunes(state, action);
       case homeScreenTypes.SUCCESS_FETCH_TRACKS:
+        console.log(action);
```

Comment:
lets remove console.logs

Reply:
we can use it for internal testings, but should not merge it in dev/staging/production